kratos/utilities/variable_utils.cpp

```diff
@@ -483,7 +483,7 @@ void VariableUtils::UpdateCurrentToInitialConfiguration(const ModelPart::NodesCo
     #pragma omp parallel for
     for (int i=0; i<num_nodes; i++) {
         const auto it_node  = nodes_begin + i;
-        noalias(it_node->Coordinates()) = it_node->GetInitialPosition();
+        noalias(it_node->Coordinates()) = (it_node->GetInitialPosition()).Coordinates();
```

Comment:
i just wonder why this is needed

Reply:
It does not compile without it (I think AMatrix is more explicit)

Reply:
but why a Point cannot be converted to an array? IT IS an array after all. @pooyan-dadvand any clue?

Reply:
It should compile....
What is the error?